AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -455,6 +465,25 @@ - (void)performOperationForPartition:(NSString *)partition
                                             }];
 }
 
+#pragma mark - MSDataStore implementation utils
+
+- (NSError *)generateDisabledError:(NSString *)operation documentId:(NSString *_Nullable)documentId {
```

Comment:
this can be a static method and be added to Utils class

Reply:
I would rename to `generateDisabledErrorWithOperation`

Reply:
I prefer to hold for moving those at the moment. There are lots of moving pieces, we don't need more conflicts.